### PR TITLE
RELATED: RAIL-5217 Use correct documentation links

### PR DESCRIPTION
**Pull request by pbenes**

### CODE REVIEW

**Comment by ghost** [comment]
Build succeeded (check pipeline).

**Review by ivanmjartan — APPROVED** [review]

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-apidocs-gate-PR19

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

File: js/generateVersionsTable.js [hunk 1]
```diff
@@ -21,13 +21,24 @@ function apiReferenceUrl(doc, version) {
...
-        version,
-        'about_gooddataui.html'
-    ];
-
-    return urlParts.filter(Boolean).join('/');
+    if (version === "next" || version === undefined) {
+        return "https://www.gooddata.com/docs/gooddata-ui/latest/";
+    } else if (version.startsWith("9")) {
+        var majorAndMinorVersion = version.split(".").slice(0, 2).join(".");
+        var corrected = majorAndMinorVersion === "9.0" ? "9.1" : majorAndMinorVersion;
```
Comment (Mara3l) [inline js/generateVersionsTable.js]: Please erase this line, as we do actually support 9.0, but it is hidden in the dropdown menu. The link should still be valid.

**Review by Mara3l — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Build succeeded (check pipeline).

**Review by Mara3l — APPROVED** [review]

**Review by ivanmjartan — APPROVED** [review]

**Review by ivanmjartan — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-apidocs-gate-PR19

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by ghost** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-sonarqube-scan-zuul-gate/30/ : SUCCESS in 1m 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-fossa-scanning-zuul/2/ : SUCCESS in 2m 04s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-merge-compliance-check/25/ : SUCCESS in 10s

**Pull request merged**